pkg/middlewares/aws_route_table_expander.go

```diff
@@ -157,3 +169,19 @@ func (m *AwsRouteTableExpander) handleDefaultTable(table *aws.AwsDefaultRouteTab
 
 	return nil
 }
+
+func (m *AwsRouteTableExpander) routeExists(routeId string, resourcesFromState []resource.Resource) bool {
+	for _, res := range resourcesFromState {
+		if res.TerraformType() != aws.AwsRouteResourceType {
```

Comment:
It may be more readable to do 

```go
for _, res := range resourcesFromState {
  if res.TerraformType() == aws.AwsRouteResourceType && res.TerraformId() == routeId {
    return true
  }
}
return false
```